### DISCUSSION TITLE
Create a dedicated "server manual"

### DISCUSSION THREAD

**Discussion by gilgongo** [jamulussoftware/jamulus#1508] [Ideas]
Information on server use is a bit scattered about on the website right now. Recent PRs have also highlighted the need for a place to put extended docs about running a server which we have parked in Tips & Tricks. The [configuration options](https://jamulus.io/wiki/Command-Line-Options) page is also getting a bit klunky. 

Over time, this isn't going to scale too well I don't think. And we have a new nav design coming in which I hope means we can have a dedicated "documentation" area.

So, along with the currently pending changes to the above, perhaps we could have:

1. A combined "Server setup" page for Win, Mac and Linux (so like a GUI server counterpart to  [Software Manual](https://jamulus.io/wiki/Software-Manual) similar to what we have [today](https://jamulus.io/wiki/Server-Win-Mac)). 
1. A bigger "Server administration manual" containing all the advanced stuff explaining what all the various options do and bringing in the content parked in Tips & Tricks. This can also have links out to our page about bandwidth and maybe server sizing advice for large ensembles.
1. Call the [Software Manual](https://jamulus.io/wiki/Software-Manual) "user manual"? 
1. Merge the [Configuration](https://jamulus.io/wiki/Command-Line-Options) page contents into the above server and client pages. Put the options common to both into an include so we can keep consistency in the wording? 

Translators note: see what I mean? :-)

**Comment by jujudusud** [comment]
Really good improvement from my point if viewer.
Actually trying to understand, as you know, who us making what... :-)
Separate in client and serveur side is the right way, I think.
